pytest.el

```diff
@@ -125,10 +135,15 @@ Optional argument FLAGS py.test command line flags."
                     (let ((testpath (if (listp tests) (car tests) tests)))
                       (pytest-find-project-root (file-name-directory testpath)))
                   (pytest-find-project-root)))
-         (tests (cond ((not tests) (list "."))
+         (tests (cond ((not tests) nil)
                       ((listp tests) tests)
                       ((stringp tests) (split-string tests))))
-         (tnames (mapconcat (apply-partially 'format "'%s'") tests " "))
+         (tnames
+          (mapconcat
+           (if pytest-source-root-dir
```

Comment:
Based on my comments above, this may need to adjust slightly to consider the boolean and what root to use when finding the relative path.